react/components/ListDetails/ItemDetails.tsx

```diff
@@ -1,21 +1,43 @@
-import { path } from 'ramda'
-import React, { Component, ReactNode } from "react"
-import { ExtensionPoint } from 'vtex.render-runtime'
-import { ButtonWithIcon, Checkbox, IconDelete } from 'vtex.styleguide'
+import React, { Component, ReactNode } from 'react'
+
+import ApolloClient from 'apollo-client'
+import classNames from 'classnames'
+import { append, map, path } from 'ramda'
+import { withApollo, WithApolloClient } from 'react-apollo'
+import { isMobile } from 'react-device-detect'
+import { InjectedIntlProps, injectIntl, IntlShape } from 'react-intl'
+import { ExtensionPoint, withRuntimeContext } from 'vtex.render-runtime'
+import {
+  ButtonWithIcon,
+  Checkbox,
+  Dropdown,
+  IconDelete,
+  withToast,
+} from 'vtex.styleguide'
 import renderLoading from '../Loading'
 
+import { getListDetailed, updateList } from '../../GraphqlClient'
+
+import wishlist from '../../wishList.css'
```

Comment:
I think it's better to name these css modules objects `styles`. It didn't immediately strike me that `wishlist` would be this.

Reply:
☝️